src/django/api/tests.py

```diff
@@ -2881,3 +2882,36 @@ def check_host(url):
         self.assertFalse(check_host(''))
         self.assertFalse(check_host(None))
         self.assertFalse(check_host('foo'))
+
+
+class RequestLogMiddlewareTest(APITestCase):
+    def setUp(self):
+        self.email = 'test@example.com'
+        self.password = 'password'
+        self.name = 'Test User'
+        self.user = User(email=self.email)
+        self.user.set_password(self.password)
+        self.user.save()
+
+        Contributor.objects.create(name=self.name, admin=self.user)
+
+    def test_request_without_token_is_not_logged(self):
+        self.client.login(email=self.email, password=self.password)
+        response = self.client.get(reverse('facility-list-list'))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(0, RequestLog.objects.all().count())
```

Comment:
This implicitly relies on the assumption that the test is operating on a fresh database. While this will mostly be true, to be a more specific test consider something like:

```python
self.assertEqual(0, RequestLog.objects.filter(user=self.user).count())
```

Reply:
Changed in 19a2eb2